client/src/components/setup/FormThirdPage.js

```diff
@@ -4,6 +4,11 @@ import * as actions from '../../actions';
 import { withRouter } from 'react-router-dom';
 
 class FormThirdPage extends Component {
+    componentDidMount() {
+        // Send an indicator that this is an update to the user model via workflow before submitting the form
+        this.props.formValues.formType = "workflow";
+    }
+
```

Comment:
Indicate to our action creator that forms coming from the workflow is indeed from the "workflow"